common/python/connectors/direct/tcs_listener/tcs_work_order_handler.py

```diff
@@ -218,11 +222,30 @@ def WorkOrderSubmit(self, **params):
             # Add to the internal FIFO
             self.workorder_list.append(wo_id)
             self.workorder_count += 1
+            # ZeroMQ for sync workorder processing
+            logger.info('TCF Workorder handler Connecting to enclave manager')
+            try:
+                socket = context.socket(zmq.REQ)
+                logger.info(socket)
+                logger.info(self.zmq_port_number)
+                socket.connect("tcp://127.0.0.1:" + self.zmq_port_number)
+                logger.info(socket)
+                logger.info('Sending request for workorder id:')
+                logger.info(wo_id)
+                socket.send_string(wo_id, flags=0, encoding='utf-8')
+                replymessage = socket.recv()
+                logger.info('Received reply from enclave')
+                logger.info(replymessage)
+                socket.disconnect("tcp://127.0.0.1:" + self.zmq_port_number)
+            except Exception as er:
+                logger.info('Exception in fetching socket')
+                logger.error(er)
 
             raise JSONRPCDispatchException(
-                WorkOrderStatus.PENDING,
-                "Work order is computing. Please query for WorkOrderGetResult \
+                WorkOrderStatus.SUCCESS,
+                "Work order has been computed. Please query WorkOrderGetResult \
                  to view the result")
```

Comment:
I'm not sure about the impact of this change to existing functionality. Till the code base is stable, safer approach would be to create a new function to handle work order synchronously/asynchronously via config file option.

Reply:
Please review the new commit.